thirdparties-extension/fr.opensagres.poi.xwpf.converter.core/src/main/java/fr/opensagres/poi/xwpf/converter/core/MasterPageManager.java

```diff
@@ -69,6 +69,10 @@
 
     private final boolean evenAndOddHeaders;
 
+    private Map<Integer, Object> headers;

```

Comment:
I think it should be better to set as final headers/footers map and initialize it in the constructor.

Reply:
The headers and footers cannot be final - see these lines:   
Map<Integer, Object> previousHeaders = headers;
Map<Integer, Object> previousFooters = footers;
headers = new HashMap<Integer, Object>();
footers = new HashMap<Integer, Object>();

Reply:
Yes you are right, please ignore my comment.